Dockerfile-production

```diff
@@ -47,7 +47,7 @@ RUN sudo apt-get install -yq nodejs \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install gem sass for grunt-contrib-sass
+# Install gem sass
 RUN gem install sass
```

Comment:
@mleanos I use NVM to switch versions. I checked on v6.3.1 and v4.5.0 and it works fine. Although when you switch versions you would have to rebuild node-sass using `npm rebuild node-sass`. But everything works fine(including sass compile to css and i see those styles applied) without need for installing ruby or sass.

I am not able to check on v0.12.0 as there is a build error. Honestly we should update node engine to remove support for those old versions.